tests/containers/podman.pm

```diff
@@ -34,6 +39,14 @@ sub run {
     my $dir = "/root/DockerTest";
     my $podman = $self->containers_factory('podman');
 
+    if ($self->firewall == 'firewalld' && !is_microos()) {
+        zypper_call('in ' . $self->firewall()) if (is_publiccloud || is_jeos);
```

Comment:
Why are we force installing firewalld on public cloud and JeOS images when it's a concious decision to not include firewalld on publiccloud or JeOS Images by default?

Shouldn't we be testing the products as we produce them..nor altering them and in the process testing a different environment than what we give to customers by default?

Reply:
Following this logic we could remove most of our test cases. Just by looking at [Tumbleweed-extra_tests_textmode](https://openqa.opensuse.org/tests/2065318) I see plenty test cases, where we first install the package and then run the regression test on it (`git`, `java`, `golang`, `sqlite3` just to name a few).

The choice to not ship `firewalld` by default on publiccloud is more related to the possibility on using the cloud-service provider firewall. This **choice** however does not mean that we can assume that nobody would use `firewalld`. In addition, `firewalld` and the container runtimes are supported packages, and the reason for not including them by default have nothing to do with the fact, that this is a legitimate and expected use case of our customers.

Also: we already had one docker-firewalld related [regression last year](https://bugzilla.suse.com/show_bug.cgi?id=1180401). IMHO is this a relevant and legitimate test case.

@pdostal was only fulfilling the requirement of [poo#94132](https://progress.opensuse.org/issues/94132), so I believe those concerns would be better addresses on the level of the PO @jlausuch

Reply:
I would have no problem with adding firewalld and testing it IN ADDITION to the default-of-no-firewall (like we do in extra tests), but this submission as it is written provides zero test coverage for networking issues on publiccloud or Jeos with no firewall, despite public cloud and Jeos with no firewall being the default

that’s not good and my concerns remain

Reply:
tell you the true I was checking this PR purely from perspective of "Containers" . The fact that we blindly taking everything written for "Containers" and running it for "Containers in Public Cloud" was not taken into consideration when I was checking this PR :( . Thanks @sysrich  for highlight this part. I second all what you are saying . I would prefer that we will exclude firewalld testing in case of "Containers in Public Cloud"

Reply:
and this actually something which was already discussed and concluded that we don't want to run firewalld in public cloud . see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13472

Reply:
I don't have a strong opinion about PC, but if the decision was to skip firewalld related topics, we should probably skip that.

For _container host_ test this PR is very much valuable. Container host testing is executed on SLE/openSUSE for both flavors, DVD and JeOS, and we are already covering non-firewall-related tests. Adding this firewall and connectivity check for me it's a plus in these flavors. SLE Micro could also benefit from this change, as firewalld comes pre-installed.

However, I could agree about uninstalling the firewall if it wasn't present by default..

Reply:
Can you please re-check the code?